script/bump-zed-minor-versions

```diff
@@ -83,29 +84,42 @@ git tag ${preview_tag_name}
 echo "Preparing main for version ${next_minor_branch_name}..."
 git checkout -q main
 git clean -q -dff
-old_main_sha=$(git rev-parse HEAD)
+git checkout -q -b ${bump_main_branch_name}
 cargo set-version --package zed --bump minor
 cargo check -q
-git commit -q --all --message "${next_minor_branch_name} dev"
+message="${next_minor_branch_name} dev
+
+Release Notes:
+
+- N/A"
+git commit -q --all --message "$message"
+
+git checkout -q main
 
 cat <<MESSAGE
-Prepared new Zed versions locally.
+Prepared new Zed versions locally. You will need to push the branches and open a PR for the change to main.
+
+# To open the PR:
+
+    git checkout ${bump_main_branch_name}
+    git push origin ${bump_main_branch_name}
+    gh pr create --fill --web
```

Comment:
1. You can avoid the `git checkout` by changing the `gh` command:
```
gh pr create --base main --head ${bump_main_branch_name}
```

2. I think there should be a single pair of commands "paste to do this" / "paste to undo". Just add the `git push origin ${bump_main_branch_name}` to the existing list of refs to push below and move the `gh pr create` command after it.